pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
Sure. Yeah, I personally think it'd be much better if they reported longitudes as negative values. But it's not extremely unusual, either.

Go ahead, add the check/enforcement if it'll make you sleep better at night. Just remember: pyoos is not necessarily ingesting data sources that follow community standards. It takes important data sources in their existing forms, and *typically* ingests and transforms them into its implementation of Unidata CDM DSG (well, except for NDBC and probably COOPS ...). Idiosyncracies are to be expected at the source.